csharp/ql/src/Telemetry/ExtractorInformation.ql

```diff
@@ -0,0 +1,195 @@
+/**
+ * @name C# extraction information
+ * @description Information about the extraction for a C# database
+ * @kind metric
+ * @tags summary telemetry
+ * @id cs/telemetry/extraction-information
+ */
+
+import csharp
+import semmle.code.csharp.commons.Diagnostics
+
+predicate fileCount(string key, int value) {
+  key = "Number of files" and
+  value = strictcount(File f)
+}
+
+predicate fileCountByExtension(string key, int value) {
+  exists(string extension |
+    key = "Number of files with extension " + extension and
+    value = strictcount(File f | f.getExtension() = extension)
```

Comment:
`count`?

Reply:
It needs to be `strictcount` otherwise `key` is not bound.

Reply:
Ah!

Reply:
Then I suppose the same holds for all the other comments.

Reply:
Yes, except the above in `fileCount`. But as `@hvitved` said that can't happen.